v2.0/app/config/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "IS_LOCAL": false,
+  "IS_LOCAL": true,
```

Comment:
we dont want to checkin is local= true right?

Reply:
Right you are.  Fixed.